tests/hooks/test_git_hook.py

```diff
@@ -121,7 +126,7 @@ def test_download_dbt_project_from_http_public_gitlab_repo(
 
 
 @pytest.mark.skipif(
-    "GITHUB_READ_TOKEN" not in os.environ,
+    not os.environ.get("GITHUB_READ_TOKEN"),
```

Comment:
nit: I prefer to check for the presence of the key, as the new change results in false positives if the Gitlab token ever happens to be a falsy value. Unlikely anyways, so I won't block merging on this.

Reply:
I changed this behavior as it was not working for me, empty variables are passed to CI even if they are not set

Reply:
Strange, well, anyways, I was being nitpicky.